src/index.js

```diff
@@ -0,0 +1,36 @@
+var express = require("express")
+var { graphqlHTTP } = require("express-graphql")
+var { graphql, buildSchema } = require("graphql")
+
+
+// construct schema
+var schema = buildSchema(`
```

Comment:
Acho que vale a pena dar uma lida sobre os [tipos de variáveis e seus respectivos usos](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)